src/status_im/contexts/onboarding/create_or_sync_profile/view.cljs

```diff
@@ -35,90 +36,115 @@
                       {:on-submit navigate-to-sign-in-by-syncing}])
      :shell?  true}]))
 
-(defn- navigate-to-sign-in-by-seed-phrase
+(defn- navigate-to-sign-in-by-recovery-phrase
   [create-profile?]
   (rf/dispatch [:syncing/clear-syncing-fallback-flow])
   (rf/dispatch [:onboarding/navigate-to-sign-in-by-seed-phrase
                 (if create-profile?
                   :screen/onboarding.new-to-status
                   :screen/onboarding.sync-or-recover-profile)]))
 
+(defn- check-your-keycard
+  []
+  (rf/dispatch [:open-modal :screen/keycard.check
+                {:on-press #(rf/dispatch [:keycard.login/check-card])}]))
+
 (defn- option-card-max-height
   [window-height]
   (- window-height
      (* 2 56) ;; two other list items
      (* 2 16) ;; spacing between items
      220)) ;; extra spacing (top bar)
 
-
-(defn- create-profile-option-card
+(defn- start-fresh-main-card
   [window-height]
   [quo/small-option-card
    {:variant             :main
-    :title               (i18n/label :t/generate-keys)
-    :subtitle            (i18n/label :t/generate-keys-subtitle)
+    :title               (i18n/label :t/start-fresh)
+    :subtitle            (i18n/label :t/start-fresh-subtitle)
     :button-label        (i18n/label :t/lets-go)
-    :accessibility-label :generate-key-option-card
+    :accessibility-label :start-fresh-main-card
     :image               (resources/get-image :generate-keys)
     :max-height          (option-card-max-height window-height)
     :on-press            navigate-to-create-profile}])
 
-(defn- sync-profile-option-card
+(defn- log-in-with-recovery-phrase-main-card
   [window-height]
   [quo/small-option-card
    {:variant             :main
-    :title               (i18n/label :t/sign-in-by-syncing)
-    :subtitle            (i18n/label :t/if-you-have-status-on-another-device)
-    :button-label        (i18n/label :t/scan-sync-code)
-    :accessibility-label :scan-sync-code-option-card
-    :image               (resources/get-image :generate-keys)
+    :title               (i18n/label :t/log-in-with-recovery-phrase)
+    :subtitle            (i18n/label :t/log-in-with-recovery-phrase-subtitle)
+    :button-label        (i18n/label :t/use-recovery-phrase)
+    :accessibility-label :log-in-with-recovery-phrase-main-card
+    :image               (resources/get-image :use-recovery-phrase)
     :max-height          (option-card-max-height window-height)
+    :on-press            #(navigate-to-sign-in-by-recovery-phrase false)}])
+
+(defn- use-a-recovery-phrase-icon-card
+  []
+  [quo/small-option-card
+   {:variant             :icon
+    :title               (i18n/label :t/use-a-recovery-phrase)
+    :subtitle            (i18n/label :t/use-a-recovery-phrase-subtitle)
+    :accessibility-label :use-a-recovery-phrase-icon-card
+    :image               (resources/get-image :ethereum-address)
+    :on-press            #(navigate-to-sign-in-by-recovery-phrase true)}])
+
+(defn- use-an-empty-keycard-icon-card
```

Comment:
Nitpick: usually articles are avoided in function names. Renamed would be `use-empty-keycard-icon-card` and `use-empty-keycard-icon-card`.